projects/Mallard/src/helpers/settings/defaults.ts

```diff
@@ -124,11 +128,13 @@ export const defaultSettings: Settings = {
     senderId: __DEV__ ? senderId.code : senderId.prod,
     isWeatherShown: true,
     wifiOnlyDownloads: false,
+    isSSR: false,
     maxAvailableEditions: 7,
     websiteUrl: 'https://www.theguardian.com/',
     logging: __DEV__
         ? 'https://editions-logging.code.dev-guardianapis.com/log/mallard'
         : 'https://editions-logging.guardianapis.com/log/mallard',
+    appsRenderingService: appsRenderingService.code,
```

Comment:
Worth a comment on why there is no Prod version

Reply:
After discussing with @groakland, this will actually need to point exclusively to PROD so that we don't require a VPN to access the endpoint. I've updated to reflect this.